productsCtrl.js

```diff
@@ -1,12 +1,23 @@
-var app = require('./index.js');
+// var app = require('./index.js');
+let db;
 
 module.exports = {
-
-  create: function(app, name, desc, price, url){
-    db.create_product([name, desc, price, url], function(err, product){
-        console.log('this is product', product)
+  setDb: function(database) {
+    db = database;
+  },
```

Comment:
because we want our callbacks on the controller to take a req and a res, i used this helper function to set the variable `db` on the controller. I pass it in in a function in the `index.js` that runs when the server starts. this might not be the best way to do it but it works. now we'll always have access to the variable `db` shouldn't be any need to pass in app